mreg_cli/network.py

```diff
@@ -203,6 +185,168 @@ def info(args):
 )
 
 
+def print_network_info(network_info: Union[str, Dict[str, Any]]) -> None:
+    """Prints info about a network given a network address string (CIDR notation),
+    or from a network info dict fetched by `get_network()`.
+
+    If a network address string is passed in, `get_network()` is called to fetch
+    information about the network with the given address.
+    """
+    if isinstance(network_info, str):
+        addr = network_info
```

Comment:
Why this `addr` variable?

Reply:
Since we are shadowing the `network_info` name 2 lines down with the result of `get_network(ip_range)` (thereby changing its type from `str` to `dict`), I added it for ease of debugging down the line should that be needed.

It has no functional impact on the code, so the block can be changed to this if needed:
```py
if isinstance(network_info, str):
    ip_range = get_network_range_from_input(network_info)
    network_info = get_network(ip_range)
```